(dp1
S'output'
p2
S"<class 'gluon.contrib.pg8000.ProgrammingError'> ('ERROR', '25P02', 'current transaction is aborted, commands ignored until end of transaction block')"
p3
sS'layer'
p4
S'/home/pedro/web2py/applications/marvel_guide/controllers/backcall.py'
p5
sS'code'
p6
S'def insert_hq():\n    data = [request.vars.hq_title,\n            request.vars.hq_description,\n            request.vars.hq_year]\n    query = """INSERT INTO hq (created, hq_title, hq_description, hq_year)\n                VALUES (\'{0}\', \'{1}\', \'{2}\', \'{3}\') RETURNING id""".format(request.now, data[0], data[1], data[2])\n    try:\n        id_hq = db.executesql(query);\n        db.commit()\n        ret = id_hq\n    except Exception as wE:\n        ret = \'erro\'\n    return db.executesql(\'Select hq_title FROM hq\',as_dict=True)\nresponse._vars=response._caller(insert_hq)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/pedro/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/pedro/web2py/applications/marvel_guide/controllers/backcall.py", line 14, in <module>\n  File "/home/pedro/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/pedro/web2py/applications/marvel_guide/controllers/backcall.py", line 13, in insert_hq\n    return db.executesql(\'Select hq_title FROM hq\',as_dict=True)\n  File "/home/pedro/web2py/gluon/packages/dal/pydal/base.py", line 1000, in executesql\n    adapter.execute(query)\n  File "/home/pedro/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1350, in execute\n    return self.log_execute(*a, **b)\n  File "/home/pedro/web2py/gluon/packages/dal/pydal/adapters/base.py", line 1344, in log_execute\n    ret = self.cursor.execute(command, *a[1:], **b)\n  File "/home/pedro/web2py/gluon/contrib/pg8000/core.py", line 573, in execute\n    self._c.execute(self, operation, args)\n  File "/home/pedro/web2py/gluon/contrib/pg8000/core.py", line 1697, in execute\n    self.handle_messages(cursor)\n  File "/home/pedro/web2py/gluon/contrib/pg8000/core.py", line 1774, in handle_messages\n    raise self.error\nProgrammingError: (\'ERROR\', \'25P02\', \'current transaction is aborted, commands ignored until end of transaction block\')\n'
p11
s.